packages/expo-document-picker/android/src/main/java/expo/modules/documentpicker/DocumentPickerModule.kt

```diff
@@ -35,7 +35,7 @@ class DocumentPickerModule : Module() {
       }
       pendingPromise = promise
       copyToCacheDirectory = options.copyToCacheDirectory
-      val intent = Intent(Intent.ACTION_OPEN_DOCUMENT).apply {
+      val intent = Intent(if (options.useGetContentAction) Intent.ACTION_GET_CONTENT else Intent.ACTION_OPEN_DOCUMENT).apply {
```

Comment:
unfortunately, the two actions are not 100% interchangeable. This will cause this issue: https://github.com/react-native-documents/document-picker/pull/557

Also, personally, I prefer not to use booleans in apis, so this could be expressed (for example) as `intentAction: 'openDocument' | `getContent'`. However, I'll leave this for others to confirm / not confirm.

Reply:
Thanks @vonovak. I'll apply the fix.

I am good with using strings too. Do we want a new enum for this in Kotlin? Or just pass a plain string from JS to Kotlin?

Reply:
Let's wait for what a code owner prefers :).